mailbox.c

```diff
@@ -533,3 +533,35 @@ void mutt_mailbox_changed(struct Mailbox *m, enum MailboxNotification action)
 
   m->notify(m, action);
 }
+
+/**
+ * email_size - Helper function to make sure we always use the same metric to
+ * compute the size of an email
+ * @param e Email
+ * @retval num Size of the email, in bytes
+ */
+static size_t email_size(const struct Email *e)
+{
+  return e->content->length + e->content->offset - e->content->hdr_offset;
```

Comment:
I think we should check that `e` and `e->content` are valid pointers just to be safe.